CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewViewController.swift

```diff
@@ -170,6 +170,14 @@ public final class ReviewViewController: UIViewController {
         return addPagesButton
     }()
 
+    private var loadingIndicatorView: UIActivityIndicatorView = {
+        let indicatorView = UIActivityIndicatorView()
+        indicatorView.hidesWhenStopped = true
+        indicatorView.style = .whiteLarge
```

Comment:
[Clarification] Will indicatorView.style = .whiteLarge work for iOS 12?

Reply:
It's the only one that works. `large` and `medium` were introduced in iOS 13.